forum/qa-include/lang/qa-lang-question.php

```diff
@@ -62,6 +62,7 @@
 		'category_ask_not_allowed' => 'You do not have permission to ask questions in this category',
 		'category_js_note' => 'To select any category, please enable Javascript in your web browser.',
 		'category_required' => 'Please choose a category',
+		'subcategory_required' => 'Please choose a subcategory',

```

Comment:
Rozumiem kwestie spójności, ale czy potrzebujemy tłumaczeń do języka innego niż Polski? Może zamiast dopisywać tłumaczenie, to lepiej usunąć ten plik? Tak jak [było to zrobione w przypadku SyntaxHighlightera](https://github.com/CodersCommunity/forum.pasja-informatyki.local/commit/beb515a21893da3748ce1db2f6da15915cac2f3a#diff-d4e8deba59503b98cce14b16bedd938dfe52bc43092c6b8ba58f9ba1847782cc)

Reply:
Jeśli chcemy usunąć to jak dla mnie należałoby usunąć wszystkie angielskie tłumaczenia z całego forum - "kwestie spójności". Prawdopodobnie można to zrobić, tzn. nie za bardzo wyobrażam sobie jak mogłyby się przydać. Można byłoby dać zmianę interfejsu per użytkownik, ale generalnie forum jest polskie, posty są po polsku, więc nie wiem czy to miałoby sens. Nie wiem też czy ma sens usuwanie tłumaczeń - trzeba by odszukać wszystkie, usunąć, sprawdzić czy wszystko jest ok. Jak dla mnie kawałek roboty która zupełnie nic nam nie da, szkoda czasu. Z rzeczami jak motywy sprawa była jasna, bo ten motyw ma modyfikacje, przeróbki pod nas, nie ma opcji przełączyć się na inny ot tak. Język w teorii można zmienić i powinno wszystko działać.